### DISCUSSION TITLE
Player and Mob shadows

### DISCUSSION THREAD

**Discussion by JupiterboyLuffy** [gri573/rethinking-voxels#151] [Ideas]
I would love to see players and mobs cast shadows!

**Comment by Pacific6938** [comment]
I think thats possible with the new raytracer now?.

**Comment by JupiterboyLuffy** [comment]
Well I use Optifine so I'll have to wait for the Alpha to support Optifine.

**Reply by Pacific6938** [reply]
Why do you use optifine if I may ask?

**Reply by JupiterboyLuffy** [reply]
because I use Forge and I like shaders so I use OptiFine and Forge so I can have my first-person mod with shaders

**Reply by DarianLStephens** [reply]
You could use Rubidium and Oculus, the ports of Sodium and Iris.
I've tried it with Rethinking Voxels, and it seems to work just fine.
